apps/modernization-ui/src/apps/page-builder/generated/models/ViewRuleResponse.ts

```diff
@@ -11,9 +11,14 @@ export type ViewRuleResponse = {
     ruleFunction?: string;
     ruleId?: number;
     sourceIdentifier?: string;
-    sourceValue?: Array<string>;
+    sourceValue?: {
+        sourceValueId: string[];
+        sourceValueText: string[];
+    };
     targetQuestions?: Array<QuestionInfo>;
     targetType?: string;
     templateUid?: number;
+    anySourceValue?: boolean;
+    targetValueText?: string[];
+    targetValueIdentifier?: string[];
```

Comment:
Issue: This appears to be a manual edit of an automatically generated file. To re-generate the `page-builder/generated` folder, run `npm run generatePageBuilder`.

You will want to make sure you have restarted the question-bank service to ensure the latest models will be created.